app/controllers/administrate/application_controller.rb

```diff
@@ -9,6 +9,7 @@ def index
       resources = apply_collection_includes(resources)
       resources = order.apply(resources)
       resources = paginate_resources(resources)
+      @resources = resources
```

Comment:
This should be available through the locals (see lines 14-19). What use case do you have where that's not enough?

Reply:
For consistency. Since I made @requested_resource and @new_resource as instance variables, I made it so that @resources can also be referenced from instance variables.